android/KMEA/app/src/main/java/com/keyman/engine/util/KMLog.java

```diff
@@ -4,35 +4,81 @@
 
 package com.keyman.engine.util;
 
+import static com.keyman.engine.KMManager.KMKey_LexicalModelID;
+
 import android.util.Log;
 import android.widget.Toast;
 
 import com.keyman.engine.BaseActivity;
 import com.keyman.engine.BuildConfig;
 import com.keyman.engine.KMManager;
-import com.keyman.engine.util.DependencyUtil;
+import com.keyman.engine.data.Keyboard;
 import com.keyman.engine.util.DependencyUtil.LibraryType;
 
+import java.util.Map;
+
 import io.sentry.Breadcrumb;
 import io.sentry.Sentry;
 import io.sentry.SentryLevel;
 
 public final class KMLog {
   private static final String TAG = "KMLog";
 
+  private static final String KEYBOARD_TAG = "keyboardId";
+  private static final String KEYBOARD_COUNT_TAG = "installedKeyboardCount";
+  private static final String MODEL_TAG = "modelId";
+  private static final String LANGCODE_TAG = "languageCode";
+
+  private static boolean isLogging = false;
```

Comment:
Why is this flag needed (as in, was this to address another problem you encountered)? It's not mentioned in the PR description.